pyasdf/generic_io.py

```diff
@@ -531,19 +531,23 @@ def read_until(self, delimiter, delimiter_name=None, include=True):
         raise ValueError("{0} not found".format(delimiter_name))
 
     def seek_until(self, delimiter, include=True):
+        last_block = b''
         while True:
             block = self.read_block()
             if not len(block):
                 break
-            index = re.search(delimiter, block)
+            blocks = last_block + block
+            index = re.search(delimiter, blocks)
```

Comment:
Yeah -- the concatenate is to deal with the case where the regular expression match falls on a block boundary.  Ideally, the Python re module would allow for this (searching through a list of strings or something), but it doesn't.